specs/protocol/stage-1.md

```diff
@@ -42,3 +44,105 @@ not be able to perform any actions which have an impact on either the safety or
 system. An example of such a role is the `SystemConfig` owner (AKA Chain Operator), which can
 modify fees and other protocol parameters. Constraints MUST be placed on the allowable range of
 values for these parameters so that they do not result in a loss of liveness or safety.
+
+## Configuration of Safes
+
+This list outlines the various Safes, their extensions, and other configuration details necessary to
+achieve Stage 1.
+
+1. **The ProxyAdminOwner Safe:** This Safe acts as the Upgrade Controller. Accordingly, it is
+   authorized to call the following safety-critical functions:
+      - All `ProxyAdmin` `onlyOwner` functions.
+      - All `DisputeGameFactory` `onlyOwner` functions.
+      - All `DelayedWeth` `onlyOwners` functions.
```

Comment:
```suggestion
      - All `DelayedWETH` `onlyOwners` functions.
```

Reply:
I consider this to be bad camel case, but it is consistent with our implementation. :)